scripts/rollup/shims/react-native/ReactFabric.js

```diff
@@ -23,6 +23,10 @@ if (__DEV__) {
   ReactFabric = require('../implementations/ReactFabric-prod');
 }
 
-BatchedBridge.registerCallableModule('ReactFabric', ReactFabric);
+if (global.RN$Bridgeless) {
```

Comment:
I'm assuming that this file gets synced over, too?

Reply:
Yes.